rpaas/plugin.py

```diff
@@ -222,6 +268,25 @@ def get_route_args(args):
     return parsed
 
 
+def get_block_args(args):
+    parser = argparse.ArgumentParser("block")
+    parser.add_argument("action", choices=["add", "list", "remove"],
+                        help="Action, add or remove blocks from nginx config")
+    parser.add_argument("-s", "--service", required=True, help="Service name")
+    parser.add_argument("-i", "--instance", required=True, help="Instance name")
+    parser.add_argument("-b", "--block_name", required=False, help="Block in nginx", type=nginx_block)
+    parser.add_argument("-c", "--content", required=False,
+                        help="(advanced) raw nginx block content")
```

Comment:
No need for the `(advanced)` notice as there's no other option. The route args only have this because it's also possible to specify a destination instead of a content.